ttnn/cpp/ttnn/operations/moreh/moreh_clip_grad_norm/moreh_clip_grad_norm_step3/device/kernels/moreh_clip_grad_norm_step3_kernel.cpp

```diff
@@ -25,18 +25,18 @@ void MAIN {
 
     // Compute cb_y
     for (uint32_t tile_idx = 0; tile_idx < num_tiles; tile_idx++) {
-        ACQ();
+        tile_regs_acquire();
         cb_wait_front(cb_x, onetile);  // comes from the reader
-        cb_reserve_back(cb_y, onetile);
```

Comment:
Same thing above

Reply:
resolved in [ce8237a](https://github.com/tenstorrent/tt-metal/pull/15897/commits/ce8237a7c45fd186600377fcd4f57975e5e5fd4e)